### PR TITLE
Add nixpkgs-update-pypi-release

### PR DESCRIPTION
**Pull request by jonringer**
Was able to run the script locally, and works as expected with a checkout of nixpkgs.

This does require a local checkout of nixpkgs though.

example result file:
```
$ cat ./package-to-update.txt | head -20
picard 2.2.3 2.3.1 https://pypi.org/project/picard/
friture 0.37 0.38 https://pypi.org/project/friture/
mopidy 2.3.1 3.0.1 https://pypi.org/project/mopidy/
tzupdate 1.5.0 2.0.0 https://pypi.org/project/tzupdate/
menumaker 0.99.11 1.4.1 https://pypi.org/project/menumaker/
buku 4.2.2 4.3 https://pypi.org/project/buku/
cozy 0.6.7 1.0 https://pypi.org/project/cozy/
guake 3.6.3 3.7.0 https://pypi.org/project/guake/
kitty 0.16.0 1.2.0 https://pypi.org/project/kitty/
udiskie 1.7.7 2.1.0 https://pypi.org/project/udiskie/
jrnl 1.9.8 2.3.1 https://pypi.org/project/jrnl/
hovercraft 2.6 2.7 https://pypi.org/project/hovercraft/
deepTools 3.3.1 3.4.1 https://pypi.org/project/deepTools/
snakemake 5.8.1 5.13.0 https://pypi.org/project/snakemake/
gallery_dl 1.12.2 1.13.3 https://pypi.org/project/gallery_dl/
truvari 1.3.2 1.3.4 https://pypi.org/project/truvari/
gnomecast 1.4.1 1.9.5 https://pypi.org/project/gnomecast/
labelImg 1.6.0 1.8.3 https://pypi.org/project/labelImg/
streamlit 0.50.2 0.57.1 https://pypi.org/project/streamlit/
luigi 2.8.11 2.8.12 https://pypi.org/project/luigi/
```

### CODE REVIEW

File: build01/nixpkgs-update.nix [hunk 1]
```diff
@@ -60,6 +62,10 @@ in
 
     serviceConfig = nixpkgsUpdateServiceConfigCommon;
```
Comment (Mic92) [inline build01/nixpkgs-update.nix]:
```suggestion
    serviceConfig = nixpkgsUpdateServiceConfigCommon;
    path = [ nixpkgs-update gnugrep ];
```

File: build01/nixpkgs-update.nix [hunk 2]
```diff
@@ -60,6 +62,10 @@ in
...
     script = ''
+      ${nixpkgs-update}/bin/nixpkgs-update delete-done
+      ${gnugrep}/bin/grep -rl ./nixpkgs -e buildPython | ${gnugrep}/bin/grep default | \
+        ${nixpkgs-update-pypi-releases} > /var/lib/nixpkgs-update/packages-to-update.txt
+      ${nixpkgs-update}/bin/nixpkgs-update update-list --cachix --outpaths
       ${nixpkgs-update}/bin/nixpkgs-update delete-done
       ${nixpkgs-update-github-releases} > /var/lib/nixpkgs-update/packages-to-update.txt
       ${nixpkgs-update}/bin/nixpkgs-update update-list --cachix --outpaths
```
Comment (Mic92) [inline build01/nixpkgs-update.nix]:
```suggestion
      nixpkgs-update delete-done
      ${nixpkgs-update-github-releases} > /var/lib/nixpkgs-update/packages-to-update.txt
      nixpkgs-update update-list --cachix --outpaths
```

File: build01/nixpkgs-update.nix [hunk 3]
```diff
@@ -60,6 +62,10 @@ in
 
     serviceConfig = nixpkgsUpdateServiceConfigCommon;
     script = ''
+      ${nixpkgs-update}/bin/nixpkgs-update delete-done
+      ${gnugrep}/bin/grep -rl ./nixpkgs -e buildPython | ${gnugrep}/bin/grep default | \
+        ${nixpkgs-update-pypi-releases} > /var/lib/nixpkgs-update/packages-to-update.txt
+      ${nixpkgs-update}/bin/nixpkgs-update update-list --cachix --outpaths
```
Comment (Mic92) [inline build01/nixpkgs-update.nix]:
```suggestion
      nixpkgs-update delete-done
      grep -rl ./nixpkgs -e buildPython | grep default | \
        ${nixpkgs-update-pypi-releases} > /var/lib/nixpkgs-update/packages-to-update.txt
      nixpkgs-update update-list --cachix --outpaths
```

File: build01/nixpkgs-update.nix (hunk 1 above)
Comment (jonringer) [reply build01/nixpkgs-update.nix]: Applied, thanks!

File: build01/nixpkgs-update.nix (hunk 2 above)
Comment (jonringer) [reply build01/nixpkgs-update.nix]: Applied, thanks!

File: build01/nixpkgs-update.nix (hunk 3 above)
Comment (jonringer) [reply build01/nixpkgs-update.nix]: Applied, thanks!

**Comment by ryantm** [comment]
Seems like it is working! Here's the first one: https://github.com/NixOS/nixpkgs/pull/83818

**Pull request merged**